minimum-cost-for-tickets.java

```diff
@@ -0,0 +1,29 @@
+// Time Complexity : O(N) where N is max element of days array + 1
+// Space Complexity : O(N)
```

Comment:
Space is constant actually ..In worst case scenario days array will have all 365 days..which is still a constant